recipes-soletta/soletta/soletta_git.bb

```diff
@@ -21,11 +22,15 @@ S = "${WORKDIR}/git"
 
 inherit cml1 python3native
 
+PACKAGECONFIG ??= "${@bb.utils.contains('DISTRO_FEATURES','x11', 'x11', '', d)}"
```

Comment:
After reading detailed the documentation you seems to be right pushing a new version :)